tests/pyreverse/test_diadefs.py

```diff
@@ -115,7 +114,6 @@ def test_extract_relations(self, HANDLER: DiadefsHandler, PROJECT: Project) -> N
         relations = _process_relations(cd.relationships)
         assert relations == self._should_rels
 
-    @pytest.mark.xfail
     def test_functional_relation_extraction(
         self, default_config: PyreverseConfig, get_project: GetProjectCallable
     ) -> None:
```

Comment:
Line 123/124 would also needs to be updated here.

Reply:
@DudeNr33 Are we sure about that?
The comment was added 14 years ago in this commit https://github.com/PyCQA/pylint/commit/7af79e602152aa122328063d5d7d5a16c65e594a

I don't think it is related to the `xfail`?

Reply:
Oh, you are right. Sorry. To be honest, if the comment is this old I have no idea what "environment problem" is really meant here.

Reply:
Yeah I also did not understand what the comment meant. Should we remove anyway ?

Reply:
I don't think so; https://en.wikipedia.org/wiki/Wikipedia:Chesterton's_fence

Reply:
This Chesterton guy must have had a lot of documentation stored in his attic 😄